### ISSUE TITLE
OpenTimeline IO : Implementation

### ISSUE THREAD

**Issue by Tilix4** [wonderunit/storyboarder#1407]
I'm currently working to an implementation of [OTIO ](https://github.com/PixarAnimationStudios/OpenTimelineIO) as universal XML exporter.

**Comment by audionerd** [comment]
I don't want to require python for Storyboarder if we can avoid it.

I know it's not recommended but I bet a simple OpenTimeline JSON export would be something we could write in JS custom to Storyboarder.

https://github.com/PixarAnimationStudios/OpenTimelineIO/blob/master/docs/tutorials/otio-file-format-specification.md

**Comment by audionerd** [comment]
Or an adapter from .storyboarder to OTIO:

https://github.com/PixarAnimationStudios/OpenTimelineIO/blob/master/docs/tutorials/write-an-adapter.md

**Comment by Tilix4** [comment]
Yes, it's another option. I proposed that because it would be easier to support several formats. To not have to write a custom exporter for every existing format or not existing yet ones.

**Comment by Tilix4** [comment]
Maybe we can propose something like : 
- Standard built-in exporter that you already have. (FCP and Premiere)
- For other less used formats, we can run python command with OTIO API, but say it requires python installed on the computer.
